gubernator/view_pr.py

```diff
@@ -94,6 +95,21 @@ def get(self, user=None):
             qs.append(ghm.GHIssueDigest.involved == user)
         prs = list(ghm.GHIssueDigest.query(*qs))
         prs.sort(key=lambda x: x.updated_at, reverse=True)
+
+        if self.request.headers.get('Accept') == 'application/json':
```

Comment:
This is cool!

Consider also adding a way to enable this behavior from the browser by changing the request URI somehow.